certbot/plugins/webroot.py

```diff
@@ -181,12 +181,8 @@ def _create_challenge_dirs(self):
                     os.chown(self.full_roots[name], stat_path.st_uid,
                              stat_path.st_gid)
                 except OSError as exception:
-                    if exception.errno == errno.EACCES:
-                        logger.debug("Insufficient permissions to change owner and uid - ignoring")
-                    else:
-                        raise errors.PluginError(
-                            "Couldn't create root for {0} http-01 "
-                            "challenge responses: {1}", name, exception)
+                    logger.debug("Unable to change owner and uid of webroot directory")
```

Comment:
My only question is whether this and the "Unable to clean up" (but not the line below) should be `logger.info`.